tasks/managed/sign-index-image/sign-index-image.yaml

```diff
@@ -203,7 +203,7 @@ spec:
             ${requestType} --pipeline "${request}" \
               -p pipeline_image="${pipeline_image}" \
               -p reference="${reference_image}" \
-              -p manifest_digest="${manifest_digest}" \
+              -p manifest_digests="${manifest_digest}" \
```

Comment:
We now support multiple components for fbc releases. And so it's no longer just one request - it's a loop now. So you should also batch the request the same as in the other tasks.

Reply:
This is now implemented in another commit.

Reply:
I believe this is done now.